audit-connector/src/main/java/com/sequenceiq/cloudbreak/audit/converter/AttemptAuditEventResultBuilderUpdater.java

```diff
@@ -0,0 +1,12 @@
+package com.sequenceiq.cloudbreak.audit.converter;
+
+import com.cloudera.thunderhead.service.audit.AuditProto;
+import com.sequenceiq.cloudbreak.audit.model.ResultEventData;
+
+public interface AttemptAuditEventResultBuilderUpdater<T extends ResultEventData> {
+
+    void updateAttemptAuditEventResultBuilderWithEventData(AuditProto.AttemptAuditEventResult.Builder builder, T source);
```

Comment:
`update` would be adequate

Reply:
`update` tells nothing about the functionality of the method. since we're not using javadocs, the method name is the only way to indicate what would the function do when somebody calls it

Reply:
well actually, your idea has gone through my head and it's not as senseless as I thought before, so I'm gonna rename it to `update` since the class name already contains a lot of information